m4_sessions/1808-inning/stress_management.md

```diff
@@ -23,10 +30,24 @@
      Share resources in your slack channel
      All sessions NEED to have a closing activity, reflection, discussion
     
-## Introductons 
+## Introductions
+* who we are
 ## Intro to new material
+* What is stress and how we manage it.
+
 ## Small Group Activity
+after introductions, we will break up into three even groups, each of which will be lead by one of us.
+* Guided Meditation *Quin
+  *Everyone in the group will be asked to lie down on a yoga mat and close their eyes. Relaxing music will be playing and I will read a guided meditation.
```

Comment:
@quinhill Will you be sure to include the meditation or a link to the meditation you are going to use?  Do you need people to bring in yoga mats?

Reply:
Quin will be sending you a link to the meditation he will be doing. He will want everyone to bring yoga mats.